weave

```diff
@@ -655,6 +655,10 @@ setup_router_iface_bridge() {
         echo "Perhaps you are running the docker daemon with container networking disabled (-b=none)." >&2
         return 1
     fi
+    # No-op if already attached
+    if netnsenter ip link show $CONTAINER_IFNAME >/dev/null 2>&1 ; then
+        return 0
+    fi
```

Comment:
I have been testing this; it looks like the `ip link set down` followed by `up` causes `weaver` to die.